marketplace/cloudinary-assets/src/components/cloudinaryAppConfig/fieldSelector.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+
+import { Form, Subheading, CheckboxField, Typography } from '@contentful/forma-36-react-components';
+
+import { ContentType, CompatibleFields, SelectedFields } from './fields';
+
+interface Props {
+  contentTypes: ContentType[];
+  compatibleFields: CompatibleFields;
+  selectedFields: SelectedFields;
+  onSelectedFieldsChange: Function;
+}
+
+export default class FieldSelector extends React.Component<Props> {
```

Comment:
Non-blocking: No reason for this to be a state component. Could just be a React function component.